src/fosslight_reuse/_result.py

```diff
@@ -212,12 +214,73 @@ def create_result_file(output_file_name, format='', _start_time=""):
     return result_file, output_path, output_extension
 
 
-def result_for_summary(oss_pkg_info_files, license_missing_files, copyright_missing_files,
-                       prj_report, _result_log, _check_only_file_mode, file_to_check_list, error_items,
-                       excluded_files, lic_present_files_in_yaml, cop_present_files_in_yaml):
+def get_only_pkg_info_yaml_file(pkg_info_files):
+    for yaml_file in pkg_info_files:
+        if yaml_file.split('/')[-1].startswith("oss-pkg-info"):
```

Comment:
How about using os.path.basename ?

Reply:
I agree with you, so I re-commit the code